bin/bootstrap.sh

```diff
@@ -22,30 +20,19 @@ LOGO="
 # Print some intro
 echo "$LOGO"
 
-
 echo "Bootstraping WebHawk Environment..."
 
 # Install pip
 pip install virtualenv
 
```

Comment:
If we are going to use pipenv maybe we do not need to install virtualenv (pipenv will take care of it).

Reply:
Missed it